FluidNC/src/Maslow/MotorUnit.cpp

```diff
@@ -212,23 +212,27 @@ bool MotorUnit::comply( double maxSpeed){
     //If the belt is moving out, let's keep it moving out
     if( distMoved > .001){
         //Increment the target
-        setTarget(positionNow + amtToMove);
+        //setTarget(positionNow + amtToMove);
         
-        amtToMove = amtToMove + 1;
+        motor.forward(amtToMove);
+
+        if(amtToMove == 0) amtToMove = 100;
+        amtToMove = amtToMove+=50;
         
-        amtToMove = min(amtToMove, maxSpeed);
-    
-    //If the belt is moving in we need to stop it from moving in
-    }else if(distMoved < -.04){
```

Comment:
We don't need this? I remember this being important to keep the belts from creeping

Reply:
That may be true. I added it a long time ago and it might not be doing anything anymore